.ci/install_footloose.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+cidir=$(dirname "$0")
+arch=$("${cidir}"/kata-arch.sh -d)
+source "${cidir}/lib.sh"
+
+if [ "$arch" != "x86_64" ]; then
+	warn "Footloose only available with x86_64."
+	exit 0
+fi
+
+footloose_repo="github.com/weaveworks/footloose"
+
+GO111MODULE=on go get "$footloose_repo"
```

Comment:
I do not think we need to install footloose itself anymore correct? We are scheduling the pod directly via k8s with all necessary bindings to get systemd working and our bats will schedule the pod, ssh into the directly and check something and exit out.  @GabyCT WDYT?